src/db/DbService.js

```
@@ -0,0 +1,179 @@
+
+        // Execute the query
+        const attestationRows = await db.query(query, queryParams);
+
+        // Return results based on `multiple` flag
+        if (multiple) {
+            return attestationRows;
+        } else {
+            return attestationRows[0] || null;
+        }
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)